recipes/main/thai_red_curry.md

```diff
@@ -0,0 +1,41 @@
+# Thai red curry
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-30%25-green.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-1.5hr-orange.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Average-yellow.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+
+**Head chef:** [@matthewdking](https://github.com/matthewdking/)
+
+**Sous chef:** [@Karyum](https://github.com/Karyum/)
+
+**Date:** 29/01/18, Week 13 FACN3
+
+**Number of servings:** 8
+
+**Author of Recipe**: [@matthewdking](https://github.com/matthewdking/)
+
+## Ingredients
+
+* 2 onions, sliced
+* 4 small sweet potatoes, peeled and chopped into about 2cm cubes
+* 1 large aubergine, chopped into about 2cm cubes
+* 2 red peppers, sliced
+* thai red curry paste
+* 3 cans of coconut milk
+* a bunch of coriander, finely chopped
+
+## Preparation
+
+1. Heat some oil in a large frying pan.
+1. Fry the onions on a medium heat until soft.
+1. Get the sweet potatoes on to boil. You don't want them completely cooked, this is just to start them off.
+1. Add the aubergine to the onions, and fry for about 10 minutes.
+1. When the aubergines are almost cooked add the red peppers.
+1. In a separate pan add some oil, and a couple of tablespoons of the paste. Heat it for a few minutes before adding a can or two coconut milk.
```

Comment:
even here you say `couple of tablespoons` like what is that ??

Reply:
you also say add 1 or 2 cans of coconut milk, but you said 3 cans in the top !! *outrages*

Reply:
Yeah I agree this is quite vague. I will change it.